Mathlib/AlgebraicTopology/SimplicialSet.lean

```diff
@@ -335,18 +338,68 @@ instance Truncated.hasColimits {n : ℕ} : HasColimits (Truncated n) := by
   dsimp only [Truncated]
   infer_instance
 
+/-- The ulift functor `SSet.Truncated.{u} ⥤ SSet.Truncated.{max u v}` on truncated
+simplicial sets. -/
+def Truncated.uliftFunctor (k : ℕ) : SSet.Truncated.{u} k ⥤ SSet.Truncated.{max u v} k :=
+  (whiskeringRight _ _ _).obj CategoryTheory.uliftFunctor.{v, u}
+
 -- Porting note (#5229): added an `ext` lemma.
 @[ext]
 lemma Truncated.hom_ext {n : ℕ} {X Y : Truncated n} {f g : X ⟶ Y} (w : ∀ n, f.app n = g.app n) :
     f = g :=
   NatTrans.ext (funext w)
 
-/-- The skeleton functor on simplicial sets. -/
-def sk (n : ℕ) : SSet ⥤ SSet.Truncated n :=
-  SimplicialObject.sk n
+/-- The truncation functor on simplicial sets. -/
+def truncation (n : ℕ) : SSet ⥤ SSet.Truncated n :=
+  (whiskeringLeft _ _ _).obj (SimplexCategory.Δ.ι n).op
 
 instance {n} : Inhabited (SSet.Truncated n) :=
-  ⟨(sk n).obj <| Δ[0]⟩
+  ⟨(truncation n).obj <| Δ[0]⟩
+
+section adjunctions
+open SimplexCategory
+
+/-- The adjunction between the n-skeleton and n-truncation.-/
+noncomputable def skAdj (n) : lan (Δ.ι n).op ⊣ truncation.{u} n := lanAdjunction _ _
+
+/-- The adjunction between n-truncation and the n-coskeleton.-/
+noncomputable def coskAdj (n) : truncation.{u} n ⊣ ran (Δ.ι n).op := ranAdjunction _ _
```

Comment:
As I am suggesting removing the abbreviation `Δ.ι`, I would suggest introducing definitions (or abbreviations) for the (co)skeleton functors.

Reply:
I'm aware of two conventions in the literature, where 
(i) `skeleton := lan (Δ.ι n).op` and where 
(ii) `skeleton := truncation.{u} n >>> lan (Δ.ι n).op`.

For the purposes of defining this adjunction, it might be useful to use definition (i) but I suspect definition (ii) would be more useful to folks interacting with simplicial sets in Mathlib. Thus my instinct is to adopt convention (ii) and just get rid of the abbreviation here (and above) as you suggest. Do you agree?